src/Compilers/CSharp/Test/Emit/CodeGen/PatternTests.cs

```diff
@@ -3165,48 +3165,45 @@ public static void Main()
 }
 ";
             var v = CompileAndVerify(source, options: TestOptions.DebugExe);
-            v.VerifyIL(qualifiedMethodName: "Program.Main",
-@"{
-  // Code size       55 (0x37)
-  .maxstack  2
-  .locals init (int V_0, //i
-                Program V_1, //y
-                Program V_2,
-                Program V_3)
-  // sequence point: {
-  IL_0000:  nop
-  // sequence point: int i = 0;
-  IL_0001:  ldc.i4.0
-  IL_0002:  stloc.0
-  // sequence point: var y = (i s ...   }).Chain()
-  IL_0003:  ldloc.0
-  IL_0004:  brfalse.s  IL_000e
-  IL_0006:  br.s       IL_0008
-  IL_0008:  ldloc.0
-  IL_0009:  ldc.i4.1
-  IL_000a:  beq.s      IL_0016
-  IL_000c:  br.s       IL_001e
-  IL_000e:  newobj     ""Program..ctor()""
-  IL_0013:  stloc.2
-  IL_0014:  br.s       IL_0026
-  IL_0016:  newobj     ""Program..ctor()""
-  IL_001b:  stloc.2
-  IL_001c:  br.s       IL_0026
-  IL_001e:  newobj     ""Program..ctor()""
-  IL_0023:  stloc.2
-  IL_0024:  br.s       IL_0026
-  IL_0026:  ldloc.2
-  IL_0027:  stloc.3
-  IL_0028:  ldloc.3
-  IL_0029:  callvirt   ""Program Program.Chain()""
-  IL_002e:  stloc.1
-  // sequence point: y.Chain2();
-  IL_002f:  ldloc.1
-  IL_0030:  callvirt   ""Program Program.Chain2()""
-  IL_0035:  pop
-  // sequence point: }
-  IL_0036:  ret
-}
+            v.VerifyIL(qualifiedMethodName: "Program.Main", @"
+    {
```

Comment:
>{ [](start = 4, length = 1)

nit: indentation unnecessary changed. It's likely due to copy/paste from TestExplorer (adds four spaces... we've reported that bug)

Reply:
I'm not willing to go through the painful hand-editing that would be required to fix this.

---
In reply to: [315403483](https://github.com/dotnet/roslyn/pull/37711#discussion_r315403483) [](ancestors = 315403483)